docs/installation/index.rst

```diff
@@ -44,3 +44,68 @@ Guides
    logging
    performance/index
    linux
+
+Post-install checklist
+----------------------
+
+After Open Zaak has been installed successfully, go through the following checklist
+to see if the software works as expected:
+
+**Check configuration**
+
+Check the configuration page for Open Zaak, accessible at the url ``https://open-zaak.gemeente.nl/view-config/``.
+This page will indicate whether certain settings are properly configured:
+
+.. figure:: assets/openzaak-config.png
+    :width: 100%
+    :alt: Open Zaak config page
+
+**Run check management commands**
+
+If Sentry was set up for Open Zaak, make sure to run the following command to
+ensure that logging to Sentry will work as expected:
+
+.. code-block:: shell
+
+    python src/manage.py raven test
+
+Furthermore, to verify that notifications can be received by the URL as defined
```

Comment:
Please add that the `test` kanaal must exist in the NRC you're using, see https://github.com/open-zaak/open-zaak/pull/442/files#r371121413

Reply:
Added a note